frontend/src/lib/main.ts

```diff
@@ -21,9 +21,13 @@ export const me: Writable<User> = writable({
 	permissions: []
 });
 
-export const branchName: Writable<string> = writable('Set Branch'); // Default branch name
+export const branchName: Writable<string> = writable('');
+export const baseBranch: Writable<string> = writable('');
 export const allBranches = writable<Branch[]>([]);
 export const editorText = writable<string>('');
+export const openIssues = writable<Issue[]>([]);
+export const selectedIssues = writable<Issue[]>([]);
+export const openPullRequests = writable<Issue[]>([]);
```

Comment:
Do these need to be global stores? Or is there a way to keep that info more localized.